homeassistant/components/recorder/migration.py

```diff
@@ -1297,8 +1342,8 @@ def migrate_context_ids(instance: Recorder) -> bool:
         is_done = not (events or states)
 
     if is_done:
-        _drop_index(session_maker, "events", "ix_events_context_id", quiet=True)
-        _drop_index(session_maker, "states", "ix_states_context_id", quiet=True)
+        _drop_index(session_maker, "events", "ix_events_context_id")
+        _drop_index(session_maker, "states", "ix_states_context_id")
 
     _LOGGER.debug("Migrating context_ids to binary format: done=%s", is_done)
     return is_done
```

Comment:
I should break this into two tasks, one for events, and one for states so the states migration isn't checking the events table every time even though its done.  But thats for another PR